client/src/components/application/DotsStepper/registrationSteps.json

```diff
@@ -0,0 +1,15 @@
+ {
+  "primaryHeader":"APPLY FOR THE GAZA CODE ACADEMY!",
+  "Info":"The Gaza Code Academy will launch its 9th cohort in August 2020.",
+  "stepHeader":"TO BE ELIGIBLE FOR THE PROGRAM, YOU MUST MEET THE FOLLOWING BASIC CRITERIA:",
+"steps":[
+"1- Be at least 18 years old and not currently enrolled at University. 2- Have strong English conversational and comprehension skills",
+"3- Have full-time availability from August 2020 to January 2021 to fully participate in the entirety of the 6-month course . 4- Have the motivation to launch a career as a developer",
+"5- Have the maturity and professionalism to treat the Code Academy like a job training (not school) . 6- Have the availability and determination to put in many hours of hard work and effort into completing the application’s technical tasks",
+"7- Have a strong love for problem-solving and being persistent in the face of challenge . 8- Have a commitment to the community by volunteering as a mentor for future Code Academy cohorts"
```

Comment:
if you didn't find expect this way, so we have to store each 2 steps in database by this way , but please try to find another way that may be better than this.